src/app/pages/questions/services/audio-record.service.ts

```diff
@@ -21,90 +17,50 @@ export class AudioRecordService {
   setAudioRecordStatus(status) {
     this.audioRecordStatus = status
   }
+
   getAudioRecordStatus() {
     return this.audioRecordStatus
   }
 
-  startAudioRecording(configFile) {
-    this.recording = this.getAudioRecordStatus()
+  startAudioRecording() {
+    this.audio = new Media(this.getPath() + this.fileName)
```

Comment:
From the documentation, it looks like you can pass callbacks for success and error scenarios. I would pass callback functions to handle these, since we do not want to get stuck on the app, if the audio task is not working.